### ISSUE TITLE
Componente botão carregar mais

### ISSUE THREAD

**Issue by erikprogramador** [vedcasts/series-casos-reais#6]
Na série casos reais já foi mostrado como fazer uma paginação utilizando uma coleção de dados vindas de um web server. Mas outra forma de realizar listagens seria um botão carregar mais, como vemos no YoutTube, na parte de videos dos canais.

![vedcasts-load-more](https://cloud.githubusercontent.com/assets/6305630/14407616/07d2395a-fea7-11e5-853b-1fee4403154e.png)

**Comment by gerardo-junior** [comment]
Acho q so usar _.concat da biblioteca lodash e o data bind do vue faz o resto.

**Comment by erikprogramador** [comment]
Sim, essa seria uma forma bem simples de implementar essa solução, porém existem nuances, por exemplo, Fazer isso utilizando web components, calcular para esconder o botão quando não houvesse mais registros, como tratar os dados vindos do web server, pois os dados podem vir em um estilo de paginação ou em uma lista ja com uma carga de dados maior, caso a lista estivesse completa teria que ser feito um calculo para colocar uma quantidade especifica de dados na lista como foi feito nas serie Vue.js.
